base/multimedia.jl

```diff
@@ -93,6 +93,12 @@ for mime in ["text/vnd.graphviz", "text/latex", "text/calendar", "text/n3", "tex
     @eval @textmime $mime
 end
 
+immutable MIMEData{mime}
+    data::Vector{UInt8}
+end
+writemime{mime <: MIME}(io::IO,::MIME"text/plain",data::MIMEData{MIME"text/plain"}) = write(io,data.data)
```

Comment:
Yeah, fair enough. This was quick and dirty, because I have to get something done. Once the name/API is agreed on I'll clean this up, export, add tests/docs, etc.